Makefile

```
@@ -114,26 +115,32 @@ test: fmt vet envtest ## Run tests.
 ##@ Build
 
 .PHONY: build
-build: fmt vet bin/vm-builder ## Build all neonvm binaries.
+build: fmt vet bin/vm-builder bin/vm-builder-generic ## Build all neonvm binaries.
 	go build -o bin/controller       neonvm/main.go
 	go build -o bin/vxlan-controller neonvm/tools/vxlan/controller/main.go
 	go build -o bin/vxlan-ipam       neonvm/tools/vxlan/ipam/main.go
 	go build -o bin/runner           neonvm/runner/main.go
 
 .PHONY: bin/vm-builder
 bin/vm-builder: ## Build vm-builder binary.
-	go build -o bin/vm-builder neonvm/tools/vm-builder/main.go
+	go build -o bin/vm-builder -ldflags "-X main.Version=${VERSION} -X main.VmInformant=${VM_INFORMANT_IMG}" neonvm/tools/vm-builder/main.go
```

```suggestion
	go build -o bin/vm-builder -ldflags "-X main.Version=${GIT_INFO} -X main.VmInformant=${VM_INFORMANT_IMG}" neonvm/tools/vm-builder/main.go
```

I think we can use `GIT_INFO` for this, like some of the other images. That way, we don't need to add `VERISON` to the makefile, and we can get a little more info :)